AntistasiOfficial.Altis/CREATE/createSupplyBox.sqf

```diff
@@ -244,7 +260,10 @@ markerSupplyCrates = markerSupplyCrates - [_marker];
 publicVariable "markerSupplyCrates";
 supplySaveArray = supplySaveArray - [[_spawnPosition, _crateType]];
 publicVariable "supplySaveArray";
+countSupplyCrates = countSupplyCrates - 1;
+publicVariable "countSupplyCrates";
 
-
-waitUntil {sleep 5; !([distanciaSPWN,1,_crate,"BLUFORSpawn"] call distanceUnits) OR (_crate distance (getMarkerPos guer_respawn) < 60)};
+systemchat format ["markerSupplyCrates END %1", markerSupplyCrates];
+systemchat format ["countSupplyCrates END = %1", countSupplyCrates];
+// waitUntil {sleep 5; !([distanciaSPWN,1,_crate,"BLUFORSpawn"] call distanceUnits) OR (_crate distance (getMarkerPos guer_respawn) < 60)};
```

Comment:
If the waitUntil is commented out, the crate will despawn instantly, that is not intended, isn´t it?

Reply:
forgot the `// DEBUG` but yeah need to be uncommented after testing is validated.